prometheus_client/exposition.py

```diff
@@ -29,7 +29,8 @@
 """Content type of the latest text format"""
 
 PYTHON26_OR_OLDER = sys.version_info < (2, 7)
-
+# Due to https://bugs.python.org/issue27657:
```

Comment:
I'd put this comment down where you use it

Reply:
Done in 41a1831